addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -168,4 +175,16 @@
 		<description>Media type of the current file</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="thumbnail" advanced="true">
+		<item-type>String</item-type>
```

Comment:
> should he change the item-type to Image?

Yes!

> The other function you describe, should that make a part of this change obsolete, is that what you mean?

If this were already implemented, it would make the change to Image type very easy for him; as it isn't yet implemented, it might be the best to wait for it (I am checking if it can be addressed in the next days).